### PR TITLE
Added a tick event to the Make a function button

### PR DESCRIPTION
**Pull request by JacobBovee**
Now we'll be able to track usage of the make a function button.

### CODE REVIEW

File: pxtblocks/blocklyloader.ts [hunk 1]
```diff
@@ -2230,6 +2230,8 @@ namespace pxt.blocks {
             workspace.registerButtonCallback('CREATE_FUNCTION', function (button) {
                 let promptAndCheckWithAlert = (defaultName: string) => {
                     Blockly.prompt(newFunctionTitle, defaultName, function (newFunc) {
+                        // Tick event to monitor usage of "Make a function" button
```
Comment (darzu) [inline pxtblocks/blocklyloader.ts]: might be redundant since the event it pretty self-explanatory

File: pxtblocks/blocklyloader.ts [hunk 2]
```diff
@@ -2230,6 +2230,8 @@ namespace pxt.blocks {
...
+                        pxt.tickEvent('blocks.make_a_function');
```
Comment (pelikhan) [inline pxtblocks/blocklyloader.ts]: there isn't a naming convention for ticks, but we never use _, can you drop them?

**Comment by darzu** [comment]
@JacobBovee , you can merge this PR whenever, maybe after you adjust to the feedback above

**Comment by pelikhan** [comment]
ping

**Pull request merged**